packages/jest-editor-support/src/Runner.js

```diff
@@ -1,49 +1,57 @@
-// @flow
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
 'use strict';
 
 const {ChildProcess} = require('child_process');
 const {readFile} = require('fs');
 const {tmpdir} = require('os');
 const {EventEmitter} = require('events');
 const ProjectWorkspace = require('./ProjectWorkspace');
-const {jestChildProcessWithArgs} = require('./JestProcess');
+const {jestChildProcessWithArgs} = require('./Process');
 
 // This class represents the running process, and
 // passes out events when it understands what data is being
 // pass sent out of the process
-
-module.exports = class JestRunner extends EventEmitter {
+module.exports = class Runner extends EventEmitter {
   debugprocess: ChildProcess;
   workspace: ProjectWorkspace;
-  jsonFilePath: string;
+  outputPath: string;
 
   constructor(workspace: ProjectWorkspace) {
     super();
     this.workspace = workspace;
-    this.jsonFilePath = tmpdir() + '/vscode-jest_runner.json';
```

Comment:
cool, this filename is only used inside this class - so it won't require changes on the integration end